README.md

```diff
@@ -1,3 +1,18 @@
 # eq-author-graphql-schema
 
 GraphQL type definitions and schema for [eq-author](https://github.com/ONSdigital/eq-author).
+
+## Publishing
+
+1. Create new branch
+2. Make any changes
+3. Commit changes
+4. Run `npm version [minor|patch|major]`. This will:
```

Comment:
Do we need to define `minor`, `patch` and `major` in the README for some guidance about which one to use?

Reply:
As per semver http://semver.org/, which npm requires